app/src/main/java/ch/epfl/sdp/blindwar/domain/game/GameSoloLocalStorage.kt

```diff
@@ -0,0 +1,29 @@
+package ch.epfl.sdp.blindwar.domain.game
+
+import android.content.ContentResolver
+import android.content.Context
+import android.content.res.AssetFileDescriptor
+import android.content.res.AssetManager
+import androidx.documentfile.provider.DocumentFile
+import java.io.FileDescriptor
+
+
+/**
+ * Game logic for the tutorial
+ *
+ * @constructor
+ * Create an instance of the tutorial's game logic
+ *
+ * @param assetManager AssetManager instance to get the mp3 files
+ */
+class GameSoloLocalStorage(
```

Comment:
Abstraction leak but it will be our task to fix it in the next sprint (time permitted)